.github/actions/setup_cloud_deps/action.yml

```diff
@@ -19,6 +19,12 @@ inputs:
   helm_password:
     description: "Harbor Registry Password"
     required: true
+  helm_version:
+    description: "Helm version to install"
+    required: false
+    # latest version at the time. v4 has breaking changes, will
+    # need to test before upgrading
+    default: "v3.19.4"
```

Comment:
3.18.4 the fallback for the setup-helm action when it can't fetch the latest release, but there are security fixes since then, so i just went to the highest 3.x.x